### DISCUSSION TITLE
Why does test flow fail in PR not main branch?

### DISCUSSION THREAD

**Discussion by enderahmetyurt** [enderahmetyurt/podiscover#200] [Q&A]
Test flow fails in its own PR branch but 
![Screenshot 2023-08-29 at 14 03 27](https://github.com/enderahmetyurt/podiscover/assets/447588/e8831582-88d1-4c61-9a7a-fe551e1dc065)

After I merged it, it doesn't fail in main branch.
![Screenshot 2023-08-31 at 10 01 24](https://github.com/enderahmetyurt/podiscover/assets/447588/93291bf5-4767-4c0b-971c-4cb95aa27775)

Could you have any idea? @kiliczsh

**Comment by kiliczsh** [comment]
I think it is related to dependabot. Can it be dependabot can't access the secrets?

**Reply by enderahmetyurt** [reply]
Wow it makes sense. I will add env variables depentabot. Thanks

**Reply by enderahmetyurt** [reply]
I added them but failure continues
![Screenshot 2023-08-31 at 10 54 51](https://github.com/enderahmetyurt/podiscover/assets/447588/989cbc05-cb03-429e-8401-21743ab4763b)

![Screenshot 2023-08-31 at 10 54 43](https://github.com/enderahmetyurt/podiscover/assets/447588/70ab3c09-802b-4be9-adad-ee8d13a7ccaf)
